### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by boutell**
Initial release of a handy utility for a pain point we just ran into.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# rename-mongodb-database
+
+<a href="https://apostrophecms.com/"><img src="https://raw.githubusercontent.com/apostrophecms/rename-mongodb-database/main/logos/logo-box-madefor.png" align="right" /></a>
```
Comment (boutell) [inline README.md]: URL should work at least in github after merge.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# rename-mongodb-database
+<a href="https://apostrophecms.com/"><img src="https://raw.githubusercontent.com/apostrophecms/rename-mongodb-database/main/logos/logo-box-madefor.png" align="right" /></a>
```
Comment (stuartromanek) [inline README.md]: Remove this badge, this is not our brand.

I know you want to formalize this but for now just use a text link to apostrophecms.com

**Review by stuartromanek — CHANGES_REQUESTED** [review]

**Comment by boutell** [comment]
Fixed that, how about now?

**Review by stuartromanek — APPROVED** [review]
cute you can pipe dump to restore

**Comment by boutell** [comment]
ja I think I nudged them toward that when I wrote pipe-friendly alternatives to mongodump and mongorestore some years ago, I saw some links to it in their internal github tickets 😄

**Pull request merged**